src/Entity/DIF.php

```diff
@@ -635,7 +635,7 @@ public function getAbstract()
      *
      * @return void
      */
-    public function setFieldOfStudyEcologicalBiological($fieldOfStudyEcologicalBiological)
+    public function setFieldOfStudyEcologicalBiological(bool $fieldOfStudyEcologicalBiological)
     {
         if ('boolean' !== gettype($fieldOfStudyEcologicalBiological)) {
```

Comment:
I guess this won't be needed anymore...

These entities are littered with checks, they should be done by Assert or Assert\Type if needed.

Reply:
Will remove in a later PR.